documentation/modules/upgrading/con-upgrade-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assemblies:
+//
+// upgrading/assembly-upgrade.adoc
+
+[id='con-upgrade-cluster-{context}']
+= Upgrading Kubernetes
+
+[role="_abstract"]
+If you are upgrading your Kubenerates cluster to a later supported version, do it before upgrading Strimzi.
```

Comment:
```suggestion
If you want to upgrade your Kubernetes cluster to a later version supported by Strimzi, do this before upgrading Strimzi.
```

Reply:
@scholzj I've changed to be less prescriptive. And added a link to the Supported versions page. (generic link attribute name so a downstream link can use the same)

![image](https://user-images.githubusercontent.com/47596553/135102276-a1425063-b751-42b3-8c52-626fdda981ea.png)

Reply:
I would leave this just at "Check the supported Kubernetes versions for your Strimzi version" and vice-versa. I do not think there is any link between the upgrades and we should not suggest that the upgrades might need to be done together or anything like that. these are really two separate things.

Reply:
This is our intro now:

![image](https://user-images.githubusercontent.com/47596553/135837994-dc4f95c5-0ecb-4a15-8fac-a3644ec1164c.png)